### ISSUE TITLE
Impossible to register model - relation already exists

### ISSUE THREAD

**Issue by cawoodm** [strapi/strapi#4609]
**Describe the bug**
When we start strapi manually (from the command line with `npx strapi start`) it seems to run fine. However when we run it as a windows service with the same command line but a different user we get errors and no server is started:

```
[2019-11-27T09:09:44.133Z] error Impossible to register the 'faqcategory' model.
[2019-11-27T09:09:44.135Z] error error: relation "faqcategories_groups" already exists
    at Connection.parseE (D:\services\strapi\node_modules\pg\lib\connection.js:606:11)
    at Connection.parseMessage (D:\services\strapi\node_modules\pg\lib\connection.js:403:19)
    at Socket.<anonymous> (D:\services\strapi\node_modules\pg\lib\connection.js:123:22)
    at Socket.emit (events.js:196:13)
    at addChunk (_stream_readable.js:290:12)
    at readableAddChunk (_stream_readable.js:271:11)
    at Socket.Readable.push (_stream_readable.js:226:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17)
From previous event:
    at Client_PG._query (D:\services\strapi\node_modules\strapi-hook-knex\node_modules\knex\lib\dialects\postgres\index.js:234:12)
    at Client_PG.query (D:\services\strapi\node_modules\strapi-hook-knex\node_modules\knex\lib\client.js:158:17)
    at Runner.query (D:\services\strapi\node_modules\strapi-hook-knex\node_modules\knex\lib\runner.js:135:36)
    at Runner.<anonymous> (D:\services\strapi\node_modules\strapi-hook-knex\node_modules\knex\lib\runner.js:228:25)
From previous event:
    at Runner.queryArray (D:\services\strapi\node_modules\strapi-hook-knex\node_modules\knex\lib\runner.js:227:12)
    at D:\services\strapi\node_modules\strapi-hook-knex\node_modules\knex\lib\runner.js:37:25
    at processImmediate (internal/timers.js:439:21)
From previous event:
    at Runner.run (D:\services\strapi\node_modules\strapi-hook-knex\node_modules\knex\lib\runner.js:25:16)
    at SchemaBuilder.Target.then (D:\services\strapi\node_modules\strapi-hook-knex\node_modules\knex\lib\interface.js:14:43)
```

**Expected behavior**
As a user I want to understand what the problem is and what I need to do to fix it. Must I delete my database, run some init script or what?

**Screenshots**
Run manually:
![image](https://user-images.githubusercontent.com/498834/69710038-e05ab500-10fe-11ea-8316-5457fc4266e3.png)

**Code snippets**
If applicable, add code samples to help explain your problem.

**System**
- Node.js version: 12.2
- NPM version: 6.9.0
- Strapi version: 3.0.0-beta.17.5
- Database: Postgres 12
- Operating system: Windows Server 2016

**Comment by derrickmehaffy** [comment]
the impossible to register means it can't connect to your database.

**Comment by cawoodm** [comment]
Doesn't "already exists" imply it can connect and finds a relation?

**Comment by lauriejim** [comment]
Hello! Do you have some steps to reproduce please?

**Comment by cawoodm** [comment]
The issue was resolved by realising that the environment for the service and for our terminal were different. Interestingly the service environment had what we thought was a correct DATABASE_USER but it was wrong.

When we ran strapi from the terminal it just worked because, even though no DATABASE_USER was defined, PostgresSQL was just implicitly trusting our user without any password and letting us connect.

I would recommend you improve your messaging: if Strapi can't connect to the DB it should just say so.
